### ISSUE TITLE
Bouncer's ghost is hidden after reaching 0hp, but doesn't die

### ISSUE THREAD

**Issue by Nightfirecat** [runelite/runelite#16547]
### Type

Incorrect behavior

### Bug description

See: https://youtu.be/ZyPP9CdffrU?t=817

I think this would reasonably be solved by the following diff:

```diff
diff --git a/runelite-client/src/main/java/net/runelite/client/game/NpcUtil.java b/runelite-client/src/main/java/net/runelite/client/game/NpcUtil.java
index b4510b22af..f742e9efc0 100644
--- a/runelite-client/src/main/java/net/runelite/client/game/NpcUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/game/NpcUtil.java
@@ -129,6 +129,8 @@ public class NpcUtil
             case NpcID.RUNITE_GOLEM:
             case NpcID.RUNITE_ROCKS:
             case NpcID.STRANGE_CREATURE_12076: // Secrets of the North transitioning to Jhallan
+            case NpcID.BOUNCER_3508:
+            case NpcID.BOUNCER_3509:
                 return false;
             // These NPCs have no attack options, but are the dead and uninteractable form of otherwise attackable NPCs,
             // thus should not be considered alive.
```

However, I'd like to solve this correctly (similar to how other transforming NPCs work--only consider their pre-transformation forms to always be considered alive, and `setDead(false)` upon transformation to keep them visible). In this case, ID 3508 is likely the form you fight, and 3509 is something related to the end of the fight? The video doesn't make clear what the difference would be here. I'd like this confirmed before adding any new cases for this NPC. This should be possible by using the NPC highlight option in the dev tools while encountering this fight in the mini-quest.

### RuneLite version

RuneLite version: 1.9.15.3

**Comment by critzd** [comment]
Went ahead and recorded the fight and 3509 is the form that is initially fought, while 3508 is the second form that you end up killing after the player goes through an animation throwing the severed leg to the 3508 version of Bouncer. Both are forms that are fought, but I imagine 3508 is related to the usage of the severed leg item.

Either way, both forms (3508 and 3509) are fought by the player.

![image](https://github.com/runelite/runelite/assets/26416138/e3bc2ee8-ae6c-4672-9333-238e1169d76e)
![image](https://github.com/runelite/runelite/assets/26416138/bde31286-61c1-48a8-ba3d-79047f187484)

**Comment by Nightfirecat** [comment]
To make sure I'm understanding you correctly, the start of the fight is against 3509, then it transforms to 3508 and you deliver the "killing blow" to that form of bouncer?

**Comment by critzd** [comment]
Yeah, that's correct.

**Comment by Nightfirecat** [comment]
Fixed in c21e5bcc0a8f3c8a0c67d5103bb6007550b1423b
